src/arrayMethodSort.js

```diff
@@ -4,8 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else if (a.toString() < b.toString()) {
+        return -1;
+      } else {
+        return 0;
+      }
+    },
+  ) {
+    let tempValue;
+
+    for (let i = 0; i < this.length; i++) {
+      for (let k = 0; k < this.length - 1; k++) {
```

Comment:
In the inner loop, you could optimize the iteration to avoid unnecessary comparisons. Since the largest element is guaranteed to be in its correct position after each pass, you can reduce the number of iterations by subtracting 'i' from 'this.length - 1'.